### PR TITLE
fix german translation

### LINKED ISSUE #21: Mixed translation
In my settings i have german as language.
But only a few strings are translated. That is no problem but the other strings are french. 

Version: 2.7
Android-version: 13
Phone: Fairphone 4

### PR DESCRIPTION
**Pull request by Stzyxh**
closes #21

but I have  few questions left because i dont really understand some strings. Could you pls explain them to me.

1. _Phone Account ID_
2. _Export fields information_ what is _fields information?_

### CODE REVIEW

**Comment by Sanmeet007** [comment]
@Stzyxh 

> **Phone Account ID**  

It refers to the SIM card slot index, used to differentiate between SIM cards in Android devices. For example, if SIM A is in slot 1 and SIM B is in slot 2, the Phone Account ID helps to identify which SIM card was used to place a call. In short helps to sort call logs based on the sim!

> **Export Fields Information**  

This refers to the fields or column headers (for CSV files) or keys (for JSON) that define the structure of the data. For example, in a CSV, "fruit" might be the header, and "banana" would be the corresponding value.

Feel free to ask any more questions!

**Comment by Stzyxh** [comment]
Ready for merge

File: lib/l10n/app_de.arb [hunk 1]
```diff
@@ -1,179 +1,178 @@
...
+  "filterText": "Filter",
+  "getStartedText": "Los geht's",
+  "ghostErrorMessage": "Hmm. Etwas ist schief gelaufen",
+  "hideText": "Ausblenden",
+  "hintMobileNumberText": "9XXXX XXXX",
+  "importCallLogsText": "Importieren Sie Ihre Anrufprotokolle",
+  "incomingText": "Eingehende",
+  "logsScreenLabelText": "Protokolle",
+  "maximumDurationHintText": "bsp: 60",
+  "maximumDurationLabelText": "Min (in secs)",
```
Comment (Sanmeet007) [inline lib/l10n/app_de.arb:67]: Instead of `Min` this should be `Max` please fix this

File: lib/l10n/app_de.arb [hunk 2]
```diff
@@ -1,179 +1,178 @@
...
+  "minimumDurationHintText": "bsp: 0",
+  "minimumDurationLabelText": "Min (in secs)",
+  "missedText": "Verpasst",
+  "mobileNumberText": "Handynummer",
+  "nextText": "Weiter",
+  "onboardingScreenFourSubtitle": "Übertragen Sie Anrufprotokolle nahtlos zwischen Geräten oder exportieren Sie sie zur sicheren Aufbewahrung und Analyse.",
+  "onboardingScreenFourTitle": "Import & Export",
```
Comment (Sanmeet007) [inline lib/l10n/app_de.arb:74]: This doesn't seems to be in German ? Google suggests me : 

```json
"onboardingScreenFourTitle:" "Importieren und Exportieren"
```
Please correct me if i am wrong. Really appreciate your work here

**Comment by Sanmeet007** [comment]
@Stzyxh I added formatting and fixed the translations above stated. Could you verify that translations are okay so i can proceed with merging this request ?

File: lib/l10n/app_de.arb (hunk 2 above)
Comment (Stzyxh) [reply lib/l10n/app_de.arb]: In german both, "Immportieren und exportieren", "import und Export " is the same

**Comment by Stzyxh** [comment]
> @Stzyxh I formatted the code and fixed the translations above as stated. Could you verify that translations are okay so i can proceed with merging this request ?

You can merge

**Pull request merged**